.changeset/bright-buckets-deliver.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-sentry': minor
+---
+
+Added annotation to specify organization per component
```

Comment:
Could you add a `**BREAKING**: ...` section as well that mentions the API change? Don't think there's a need to go into details though, just the fact that it changed.

Reply:
Done!